src/components/LinkedAccounts/LinkedAccountsContent.tsx

```diff
@@ -35,22 +35,132 @@ export const LinkedAccountsContent = ({
 
   return (
     <div className='Layer__linked-accounts__list'>
-      {data?.map((account, index) => (
-        <LinkedAccountOptions
-          key={`linked-acc-${index}`}
-          config={linkedAccountOptionsConfig}
-          accountId={account.id}
-          connectionId={account.connection_id}
-          source={account.external_account_source}
-          showLedgerBalance={showLedgerBalance}
-        >
-          <LinkedAccountThumb
-            account={account}
-            asWidget={asWidget}
+      {data?.map((account, index) => {
+        let pillConfig
+        if (account.requires_user_confirmation_as_of) {
+          pillConfig = {
+            text: 'Confirm account',
+            config: [
+              {
+                name: 'Oops... this is a duplicate account!',
+                action: async () => {
+                  // TODO: trigger some sort of loading spinner here
```

Comment:
Looking for your opinion: is this something we should do right now? Can bring in design help if so.